spec/victor-cli/commands/generate_spec.rb

```diff
@@ -23,11 +23,12 @@
   end
 
   context "with SVG_FILE RUBY_FILE" do
+    let(:ruby_file) { 'spec/tmp/code.rb' }
+    before { system "rm #{ruby_file}" if File.exist? ruby_file }
```

Comment:
Perhaps `File.unlink ruby_file` is a bit cleaner. I think it will raise an exception if a failure occurs which might be better than using system.

Reply:
Done.